doc/howtos/image-installer/init-tpm

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -ex
```

Comment:
I don't have a strong opinion about the location of these scripts. I can move them.